<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="reset_button">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_phase1">
           <property name="text">
            <string>Phase 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbox_phase1">
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_phase1">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_phase2">
           <property name="text">
            <string>Phase 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbox_phase2">
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_phase2">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_phase3">
           <property name="text">
            <string>Phase 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbox_phase3">
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_phase3">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_phase4">
           <property name="text">
            <string>Phase 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbox_phase4">
           <property name="maximum">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_phase4">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
